_pages/plugins/adipoq.md

```diff
@@ -0,0 +1,28 @@
+---
+mediawiki: AdipoQ
+title: AdipoQ
+description: A simple toolbox of two ImageJ plugins for quantifying adipocyte morphology and function in tissues and in vitro.
+icon: https://raw.githubusercontent.com/hansenjn/AdipoQ/main/Webfiles/AdipoQ%20Logo.png
+categories: [Automation, Analysis, Particle Analysis, Segmentation, Object Detection, Colocalization]
+doi: 10.1091/mbc.E21-11-0592
+source-url: https://github.com/hansenjn/AdipoQ
+dev-status: Stable
+support-status: Active
+team-founder: '@hansenjn'
+team-maintainer: '@hansenjn'
+---
+
+AdipoQ is a simple toolbox of two ImageJ plugins for quantifying adipocyte morphology and function in tissues and in vitro.
+
+The software is published with the following publication:
+Katharina Sieckmann, Nora Winnerling, Mylene Huebecker, Philipp Leyendecker, Dalila Juliana Silva Ribeiro, Thorsten Gnad, Alexander Pfeifer, 
+Dagmar Wachten, Jan N. Hansen. *AdipoQ—a simple, open-source software to quantify adipocyte morphology and function in tissues and in vitro.* 
+Molecular Biology of the Cell 2022, 33:12. doi: https://doi.org/10.1091/mbc.E21-11-0592
```

Comment:
Please use the `{% include citatation}` mechanism to cite publications. See https://imagej.net/editing/citations